packages/client/hmi-client/src/components/widgets/tera-input.vue

```diff
@@ -4,6 +4,7 @@
 		<main :class="{ error: getErrorMessage }" @click.self.stop="focusInput">
 			<input
 				ref="inputField"
+				v-bind="$attrs"
```

Comment:
what is this change for?

Reply:
This is so I can pass things such as "disabled" to this component
This was @shawnyama's change